docs/adr/0023-paypal-support.md

```diff
@@ -0,0 +1,104 @@
+# Add PayPal as a supported payment option
+
+- Deciders: Ben Bangert, Barry Chen, Tyler Duzan, Bianca Danforth, Se Yeon Kim, Wennie Leung
+- Date: 2020-11-11
+
+## Context and Problem Statement
+
+The Subscription Platform was implemented with Stripe as the payment processor and driver of subscription logic. Stripe is used as the source of truth for whether an account has subscriptions associated with it. We would like to add PayPal as a payment option for subscriptions, and integrate it appropriately with the existing system in a least-effort manner that is ideally flexible enough to support future expansions of payment processors, such as IAP.
+
+## Decision Drivers
+
+- Engineering resources
+- Future effort needed to integrate IAP
+- Code complexity as it relates to existing Subscription Platform infrastructure
+
+## Considered Options
+
+- Stripe-driven out-of-band invoice processing
+- PayPal subscriptions linked in FxA
+- Recurly
+
+## Decision Outcome
+
+Originally considered to be the 'fast-path', using PayPal subscriptions ended up being approximately the same engineering effort to implement as Stripe-driven invoice processing. Given additional user experience benefits of the Stripe-driven approach, it is now the recommend implementation path.
```

Comment:
General ADR nit: Maybe lead with literally "Stripe-driven out-of-band invoice processing" to match up with the chosen option at a glance. Fine to elaborate after that, but best to very clearly highlight the decided option up front.